packages/vsce-api/src/interfaces/IExtensionAPI.ts

```diff
@@ -10,9 +10,14 @@
  */
 
 import { ResourceTypes } from "../resources";
+import { Event } from "vscode";
+import { IResourceInspectEvent } from "./IResourceInspectEvent";
 
 export interface IExtensionAPI {
   resources: {
     supportedResources: ResourceTypes[];
+  },
+  events: {
+    onDidInspectResourceEvent: Event<IResourceInspectEvent>;
```

Comment:
Convention seems to be this kind of naming convention, like the built in `onDidChangeTreeData` - not suffixed with `Event` - sorry to be pedantic 
```suggestion
    onDidInspectResource: Event<IResourceInspectEvent>;
```